src/modules/categories/components/category/category.jsx

```diff
@@ -41,7 +41,7 @@ export default class Category extends Component {
     const s = this.state
 
     const isNullCategory = p.category === 'null-category' && p.popularity === 0
-    const roundedPop = new BigNumber(p.popularity).integerValue(BigNumber.ROUND_HALF_EVEN)
+    const roundedPop = new BigNumber(p.popularity.toString()).integerValue(BigNumber.ROUND_HALF_EVEN)
     let popString = roundedPop.toNumber() === 1 ? ' SHARE' : ' SHARES'
     if (roundedPop > 1000) {
       const thousands = roundedPop / 1000
```

Comment:
Don't use JS numbers!  If you divide a BigNumber by a JS number, the output is a JS number.

Reply:
existing code, will rebase to see if fixed as part of bignumber changes. if not will fix.